lib/models/device.coffee

```diff
@@ -201,6 +201,30 @@ exports.isOnline = (uuid, callback) ->
 	exports.get(uuid).get('is_online').nodeify(callback)
 
 ###*
+# @summary Get the local IP address of a device
+# @name getLocalIPAddress
+# @public
+# @function
+# @memberof resin.models.device
+#
+# @param {String} uuid - device uuid
+# @returns {Promise<String|undefined>} local ip address
+#
+# @throws Will throw if the device is offline.
+#
+# @example
+# resin.models.device.getLocalIPAddress('7cf02a62a3a84440b1bb5579a3d57469148943278630b17e7fc6c4f7b465c9').then (localIPAddress) ->
+# 	console.log(localIPAddress)
+###
+exports.getLocalIPAddress = (uuid, callback) ->
+	exports.get(uuid).then (device) ->
+		if not device.is_online
+			throw new Error("The device is offline: #{uuid}")
+
+		return device.ip_address?.split(' ')[1]
```

Comment:
This is not a sensible way to do it, there is no guarantee on the ordering of the ip addresses, nor that the device will only have 1 local ip address.  The better way to do this is to remove only the ip address that matches `vpn_address`